config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml

```diff
@@ -1,37 +1,6 @@
 # kubeadm upgrade tests
 periodics:
 # manual-release-bump-required (create a new job and delete the oldest)
-- name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-11-1-12
```

Comment:
we planned to clean all kubeadm*gce in a batch, but this one is OK to be removed in this PR.
[1]

Reply:
clean = rewrite or replace. I see some kubeadm-kinder upgrade jobs. Are they complimentary or replacement for this one?

Reply:
replacements.